Containerfile.rag

```diff
@@ -1,21 +1,18 @@
 # vim: set filetype=dockerfile
-
-# port, commands, and other details inherited from the base image
 ARG OLS_VERSION=latest
+ARG OCP_VERSION=4.15
+ARG LIGHTSPEED_RAG_CONTENT_DIGEST=sha256:eb3472dd675cc79b02e4d15b53b12c9c365d6c2335a6026f55de443dd95c902f
+
+FROM quay.io/openshift/lightspeed-rag-content@${LIGHTSPEED_RAG_CONTENT_DIGEST} as lightspeed-rag-content
+ARG OLS_VERSION
```

Comment:
i don't understand the purpose of this line?

Reply:
Generatlly that's passing an ARG from one stage to another, but, true, not needed in this case.

Reply:
Done